src/sbvr-api/sbvr-utils.ts

```diff
@@ -1386,6 +1232,7 @@ export const getAffectedIds = Promise.method(
 				url: `/${request.vocabulary}${request.url}`,
 			})
 			.then(request => {
+				request.engine = db.engine;
```

Comment:
Previously when we were all in the same file everywhere could just access the module level `db` instance, with it being split into a separate file I changed things to access the `engine` property from the request instead rather than propogating semi-global style variables around modules